sympy/physics/mechanics/particle.py

```diff
@@ -249,7 +249,7 @@ def __calc_gforce(self, p):
         """
 
         vect_dist = p.point.pos_from(self.point)
-        return ( (self.mass * p.mass) / vect_dist.magnitude() ** 2) * vect_dist.normalize()
+        return ( (self.mass * p.mass * G) / vect_dist.magnitude() ** 2) * vect_dist.normalize()
```

Comment:
I think this would be faster as
```suggestion
        return ( (self.mass * p.mass * G) / vect_dist.magnitude() ** 3) * vect_dist
```

Reply:
okay sir